README.md

```diff
@@ -2,6 +2,7 @@
 
 [![Project Status: Active – The project has reached a stable, usable state and is being actively developed.](https://www.repostatus.org/badges/latest/active.svg)](https://www.repostatus.org/#active)
 [![CI](https://github.com/nginx/unit/actions/workflows/ci.yml/badge.svg)](https://github.com/nginx/unit/actions/workflows/ci.yml "GitHub workflow CI")
+[![Discussions](https://img.shields.io/badge/N_Unit-discussions-green)](https://github.com/nginx/unit/discussions "NGINX Unit Discussion Board")
```

Comment:
Please make the alt text `GitHub Discussions`

I think it would be better if the badge text read `NGINX Unit` rather than 'N Unit'.

Also the 'green' seems a little off, interestingly if you chop off the 'n' (and more) from green you get a better colour, e.g

<https://img.shields.io/badge/N_Unit-discussions-gree>

Reply:
Thanks for finding the "closer" color. Not sure why, but it seems equivalent to https://img.shields.io/badge/N_Unit-discussions-g 

The suggested alt text makes sense.

Reply:
Hmm, actually I'm wondering if `GitHub | discussions` would make more sense than `NGINX Unit | discussions`?

What do other projects use?

Reply:
@ac000 , from a UI Text PoV, we'll see this badge _only_ in the README for this well-labeled NGINX repo. So I think it's OK to label this `GitHub | discussions`.